File: 657.c
bool judgeCircle(char* moves) {
    int i= 0;
    int down =0, left = 0;
   while(moves[i]!='\0')
    {
        switch (moves[i])
        {
            case 'U':   down++;break;
            case 'D':   down--;break;
            case 'L':   left++;break;
            case 'R':   left--;break;
        }    
       i++;        
    }
    if (down==0&&left==0 )
        return true;
    else 
        return false;
}

File: Python/657.py
class Solution(object):
    def judgeCircle(self, moves):
        """
        :type moves: str
        :rtype: bool
        """
        x=0
        y=0
        for step in moves:
            if(step == 'L'):
                x-=1
            elif(step == 'R'):
                x+=1
            elif(step == 'U'):
                y+=1
            elif(step == 'D'):
                y-=1
        if(x==0 and y == 0):
            return True
        else:
            return False
res = Solution()
res.judgeCircle("LLRRUUDD")
if(res):
    print("true")
else:
    print("false")